File: src/output/exit-codes.ts
export const EXIT_CODES = {
  SUCCESS: 0,
  NO_CHANGE: 0,
  DRY_RUN: 0,
  USER_CANCEL: 130,
  ERROR: 1
} as const;

export type ExitCode = (typeof EXIT_CODES)[keyof typeof EXIT_CODES];

File: internal/output/exit_codes.go
package output

type ExitCode int

const (
	Success    ExitCode = 0
	NoChange   ExitCode = 0
	DryRun     ExitCode = 0
	UserCancel ExitCode = 130
	Error      ExitCode = 1
	UsageError ExitCode = 2
)
